docs/docs/features/garbage-collector.md

```
@@ -17,7 +13,7 @@ The “Enable GC Logging” option allows you to do this.
 ---
 
 
-## Garbage Collector Details{#details}
+## Garbage Collector Details
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)